upgrade/overview.md

```diff
@@ -33,7 +33,7 @@ You can use a [manual installation approach]({%slug installation-approaches%}#ma
 
 If you have purchased a license and you need to migrate from the trial package to the licensed commercial version do the following steps:
 
-***Using Assemblies**
+**Using Assemblies**
```

Comment:
Replace the text in bold with items in an unordered list. Then indent each of the numbered lists so they get nested under the corresponding unordered list item. For example:

* When using assemblies:
   1. Make sure you have removed the trial version first.
   1. Install the commercial version.
   1. Replace the references in the project.
       [...]
* When using the Telerik NuGet Feed:
   1. ...